src/Components/WebAssembly/testassets/HostedInAspNet.Client/Pages/Home.razor

```diff
@@ -0,0 +1,3 @@
+﻿@page   "/"
```

Comment:
Instead of modifying the `HostedInAspNet.Client` project, you can create a new test page in the `BasicTestApp`.

Check out this directory for examples of other test cases: https://github.com/dotnet/aspnetcore/tree/cfae382c2934814910d9bf1154e06d4a6f141ed5/src/Components/test/testassets/BasicTestApp

Reply:
Thanks for the reviewing @captainsafia 



![image](https://user-images.githubusercontent.com/18040506/96916711-ee7f9580-1475-11eb-8ec1-25ee95da7b96.png)

I can't replicate the bug in the `BasicTestAPP` because it is an standalone app and the navigation interception is off.

There is another server hosted project to create the test page?

Reply:
Ah, yes -- we support running the `BasicTestApp` under different hosts by providing the startup logic. You can see an example of this in the AuthTests:

https://github.com/dotnet/aspnetcore/blob/63199b40afd17586ee25d4ab29a42af5058edac9/src/Components/test/E2ETest/Tests/AuthTest.cs#L34-L55

We have some helper functions that allow you to configure an entire test case with this model as seen here:

https://github.com/dotnet/aspnetcore/blob/8c8cb4dce235b099d1e419b6a039217eb56a77e6/src/Components/test/E2ETest/Tests/ProtectedBrowserStorageInjectionTest.cs#L18-L25